src/bin/muxer/mod.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2017-2018, The rav1e contributors. All rights reserved
+//
+// This source code is subject to the terms of the BSD 2 Clause License and
+// the Alliance for Open Media Patent License 1.0. If the BSD 2 Clause License
+// was not distributed with this source code in the LICENSE file, you can
+// obtain it at www.aomedia.org/license/software. If the Alliance for Open
+// Media Patent License 1.0 was not distributed with this source code in the
+// PATENTS file, you can obtain it at www.aomedia.org/license/patent.
+
+mod ivf;
+use self::ivf::IvfMuxer;
+
+mod mp4;
+use mp4::Mp4Muxer;
+
+mod y4m;
+pub use self::y4m::write_y4m_frame;
+
+#[cfg(feature = "ffmpeg-sys")]
+mod avformat;
```

Comment:
What is `avformat` used for in this file?

Reply:
sorry for miss this. the mp4.rs depends on avformat, so  I need this to export avfrormat.rs for mp4.rs